dcpy/connectors/edm/recipes.py

```diff
@@ -130,7 +154,14 @@ def import_dataset(
         )
         if preprocessor is not None:
             df = preprocessor(ds.name, df)
+
+        # make column names more sql-friendly
+        columns = {
+            column: column.replace("-", "_").replace("'", "_") for column in df.columns
+        }
+        df.rename(columns=columns, inplace=True)
         pg_client.insert_dataframe(df, ds_table_name)
+        pg_client.add_pk(ds_table_name, "ogc_fid")
```

Comment:
Figured it made more sense on the recipe import side to specify "ogc_fid" as id column rather than in the pg util, keep the code in utils a bit more pure